src/Tus/Server.php

```diff
@@ -486,6 +489,17 @@ protected function verifyPatchRequest(array $meta) : int
             return HttpResponse::HTTP_CONFLICT;
         }
 
+        $contentType   = $this->request->header('Content-Type');
+        $contentLength = $this->request->header('Content-Length');
+
+        if ( ! $contentType || $contentType !== self::HEADER_CONTENT_TYPE) {
```

Comment:
first check `! $contentType` is not necessary, second check will also be false in case `$contentType` is null or empty?